webapp/src/main/resources/properties/en.properties

```diff
@@ -718,6 +718,12 @@ workbench.translationHistoryModal.title=Translation History
 # Label for the button to close the translation history modal dialog
 textUnit.translationHistoryModal.close=Close
 
+# Text telling the user why one of the rows is highlighted
+textUnit.translationHistoryModal.currentUnitExplanation=The highlighted row is the currently approved translation.
+
+# Text explaining why the currently approved translation is not the latest
+textUnit.translationHistoryModal.currentNotLatest=In some cases, the currently approved translation is not the most recent one.
```

Comment:
ditto

Reply:
I guess in that case we can be specific since it should not show often: "Some edits happened in parallel: more recent translations from a drop were imported but not set as current translation."